meson-post-install.py

```diff
@@ -7,13 +7,12 @@
 datadir = os.path.join(prefix, 'share')
 
 # Packaging tools define DESTDIR and this isn't needed for them
-if not 'DESTDIR' in os.environ:
-	print('Compiling gsettings schemas...')
-	subprocess.call(['glib-compile-schemas', os.path.join(datadir, 'glib-2.0', 'schemas')])
+if 'DESTDIR' not in os.environ:
```

Comment:
How is this different from `if not 'DESTDIR' in os.environ:`?

Reply:
In Python PEPs (Python Enhancement Proposals), it is recommended to use the "if x not in y" syntax instead of "if not x in y" for better readability and consistency with other programming languages.

Here are a few reasons why "if x not in y" is considered better:

1. Readability: The "not in" operator reads more naturally and intuitively, making the code easier to understand for both experienced and new Python developers.

2. Consistency: Many other programming languages use the "not in" syntax, so using it in Python promotes consistency across different codebases and reduces confusion when switching between languages.

3. Negation clarity: Placing the negation ("not") before the containment check ("in") makes it clear that we are checking for absence rather than presence. This can prevent potential logical errors or misunderstandings.

4. Operator precedence: The "not in" operator has higher precedence than "in", which can help avoid unintended behavior when combining multiple conditions or operators within an if statement.

Overall, using "if x not in y" aligns with Python's philosophy of emphasizing code readability and reducing ambiguity, making it a preferred choice according to PEP guidelines.